### DISCUSSION TITLE
Do I need to update code signing identities after adding new device on apple developer portal

### DISCUSSION THREAD

**Discussion by mouricef** [codemagic-ci-cd/codemagic-docs#1451] [Q&A] [answered]
Hi All,

I have setup iOS build successfully on code magic using code signing identities from Team settings. 

I have metioned the profile and certificate inside the yml file using the reference name like below,

ios_signing:
                provisioning_profiles:
                    - xxxxxx_codemagic_dev
                    - xxxxxx_codemagic_widgetextension_dev
                    - xxxxxx_codemagic_notificationextension_dev
                certificates:
                    - xxxxxx_codemagic_development

My question is, If I add new device from apple developer portal to the existing profile used above. Do I have to again fetch the profile inside codemagic Team setting?

**Comment by icarusdust** [comment, marked as answer]
Hi @mouricef, yes you will need to do it, otherwise you will continue using the old profile that you uploaded manually.
